src/frontend/src/components/DesignReviewDelayModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { TextField, Button, Link } from '@mui/material';
+import { useState, ChangeEvent } from 'react';
+import { DesignReview, wbsPipe } from 'shared';
+import { useToast } from '../hooks/toasts.hooks';
+import { routes } from '../utils/routes';
+import NERModal from './NERModal';
+import { Link as RouterLink } from 'react-router-dom';
+
+// delay wp modal component
```

Comment:
you don't need these comments, the name of the component/ placement of the file in our file structure should be enough explanation for what the component is/ how it's used